webapp/src/Controller/Jury/ConfigController.php

```diff
@@ -75,6 +75,12 @@ public function indexAction(EventLogService $eventLogService, Request $request):
                         }
                     }
                     $data[substr($key, strlen('config_'))] = $valueToUse;
+                    if (strpos($key, 'config_lazy_eval_results') === 0 && $value !== '2') {
+                        $this->dj->unblockJudgeTasks();
+                        var_dump("unlock");
+                    } else {
+                        var_dump("lock");
```

Comment:
Do we want to lock something? Probably finishing the current set is fine?

Reply:
This was debug for what happens if you change the lazy from 2 to something else. In this case we create all missing queue tasks but as I before started with an extra boolean which sometimes was not sent on post (so it can be easily hidden when you're not shadow to prevent people from selecting the `on demand` when they don't know what it does)

But all that is now removed so this was already wrong debugging stuff.